mParticle-AppsFlyer.podspec

```diff
@@ -15,10 +15,10 @@ Pod::Spec.new do |s|
     
     s.static_framework = true
 
-    s.ios.deployment_target = "8.0"
+    s.ios.deployment_target = "9.0"
     s.ios.source_files      = 'mParticle-AppsFlyer/*.{h,m,mm}'
     s.ios.dependency 'mParticle-Apple-SDK/mParticle', '~> 7.0'
-    s.ios.dependency 'AppsFlyerFramework', '~> 5.0'
+    s.ios.dependency 'Beta-AppsFlyerFramework', '~> 6.0'
     s.ios.pod_target_xcconfig = {
         'FRAMEWORK_SEARCH_PATHS' => '$(inherited) $(PODS_ROOT)/AppsFlyerFramework/**',
```

Comment:
Does this line need to be updated to use Lib instead of Framework (or otherwise?)